ghost/admin/app/controllers/tags.js

```diff
@@ -16,9 +16,18 @@ export default class TagsController extends Controller {
     }
 
     get filteredTags() {
-        return this.tags.filter((tag) => {
-            return (!tag.isNew && !tag.isDestroyed && !tag.isDestroying && !tag.isDeleted && (!this.type || tag.visibility === this.type));
+        // new tags are preemptively added to tagsScreenInfinityModel,
+        // but if the user hasn't fetched them on /tags yet they'll show up twice
+        // this makes sure each tag only shows up once
+        const tagMap = new Map();
+        
+        this.tags.forEach((tag) => {
+            if (!tag.isNew && !tag.isDestroyed && !tag.isDestroying && !tag.isDeleted && (!this.type || tag.visibility === this.type)) {
+                tagMap.set(tag.id, tag);
+            }
         });
+        
+        return [...tagMap.values()];
```

Comment:
This seems unnecessary now with the fix for the incorrect push onto the array. Was there something else that prompted this to be included or was it from an earlier fix attempt?